planner-api/src/main/java/pl/edu/agh/kuce/planner/auth/Current.java

```diff
@@ -0,0 +1,13 @@
+package pl.edu.agh.kuce.planner.auth;
+
+import org.springframework.security.core.annotation.CurrentSecurityContext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
```

Comment:
I think it would be great to add a comment here on what exactly that lines do. For me it wasn't obvious at first and even after reading about them, I am not 100% sure what exactly happens here.